var/spack/repos/builtin/packages/vtk-m/package.py

```diff
@@ -31,26 +31,26 @@ class VtkM(CMakePackage, CudaPackage, ROCmPackage):
     version("release", branch="release")
```

Comment:
There doesn't appear to be a `release` branch.  Do you want to add it or remove the `version` directive here?

Reply:
Hmm. How hard would it be to have the `release` branch mirrored on GitHub?

Reply:
I'm looking into this.

Reply:
Ok, we should be syncing the `release` branch now too.

https://github.com/Kitware/VTK-m/tree/release

Reply:
Works for me.  Thanks!